src/lib/__tests__/utils-test.js

```diff
@@ -342,6 +342,45 @@ describe('utils.getRefinements', () => {
     );
   });
 
+  it('should retrieve one facetRefinement and inject query facet if clearQuery === true', () => {
```

Comment:
Since we already cover the case with facet refinements, we can focus the test only on query it will simplify it a bit.